src/ir/auth_logic/BUILD

```diff
@@ -90,3 +92,26 @@ cc_test(
         "//src/common/testing:gtest",
     ],
 )
+
+#antlr
+
+antlr(
+    name = "auth_logic_antlr",
+    srcs = ["//rust/tools/authorization-logic:authlogic_grammar"],
+    language = "Cpp",
+    listener = False,
+    package = "",
+    visitor = True,
+)
+
+cc_library(
```

Comment:
Does this rule produce the CPP code for both the visitor and the parser or just the parser? If it produces both the visitor for the parser and the parser, is it better to give it a name that expresses that it produces both of these, like `auth_logic_antlr_generated_cc` or something like that? If it is really just the parser, then the current name probably makes sense.

Reply:
(And this is just a suggestion that you can ignore if you prefer the current name)

Reply:
Yes, this produces both visitor and parser. I made a change here replacing 2 rules (antlr and cc_library) with a single rule antlr4_cc_combined.